ci/travis/ci.sh

```diff
@@ -253,10 +258,16 @@ install_ray() {
   (
     cd "${WORKSPACE_DIR}"/python
     build_dashboard_front_end
-    keep_alive pip install -v -e .
+    keep_alive pip install -v -v -e .
   )
 }
 
+uninstall_ray() {
```

Comment:
Is this used somewhere?

Reply:
Might be left over from debugging, I'll check.